### PR TITLE
LPS-84376 Add build files

### PR DESCRIPTION
**Pull request by pavel-savinov**

### CODE REVIEW

**Comment by brianchandotcom** [comment]
@pavel-savinov please resend, since I didn't get an answer yet, I'm closing the ticket. See my comment above. Thx.

File: modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/internal/security/permission/resource/AssetListEntryPortletResourcePermission.java [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+)
+public class AssetListEntryPortletResourcePermission
+	implements PortletResourcePermission {
+
+	@Override
+	public void check(
+			PermissionChecker permissionChecker, Group group, String actionId)
+		throws PrincipalException {
+
+		if (!contains(permissionChecker, null, actionId)) {
```
Comment (Preston-Crary) [inline modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/internal/security/permission/resource/AssetListEntryPortletResourcePermission.java]: Why are you using null here for a GroupedModel?  I think the only ones of these that should be passing null in is models without groupIds.  Shouldn't you just use the factory with the built in behavior?

File: modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/service/base/AssetListEntryLocalServiceBaseImpl.java [hunk 2]
```diff
@@ -0,0 +1,716 @@
...
+		assetListEntry = assetListEntryPersistence.findByPrimaryKey(assetListEntry.getPrimaryKey());
+
+		AssetListEntryLocalization assetListEntryLocalization = assetListEntryLocalizationPersistence.fetchByAssetListEntryId_LanguageId(assetListEntry.getAssetListEntryId(),
+				languageId);
+
+		return _updateAssetListEntryLocalization(assetListEntry,
+			assetListEntryLocalization, languageId, title, description);
+	}
+
+	@Override
```
Comment (Preston-Crary) [inline modules/apps/asset/asset-list-service/src/main/java/com/liferay/asset/list/service/base/AssetListEntryLocalServiceBaseImpl.java]: Is there anything different in the behavior of this method then what you guys are doing?  I know the it's a Map<String, String> but this is a new service, you should be able to create the map and pass it down to the service.

**Comment by jkappler** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**